web/docs/operators/package.md

```diff
@@ -68,3 +74,28 @@ Remove the installed package `zeek`:
 ```
 package remove zeek
 ```
+
+## Package List Formats
+
+The output of the `packages` operator can be controlled by the `--format`
+parameter
+
+### Compact Format
+
+The `compact` format consists of one object per installed package
+with keys `id`, `name`, `author` and `description`.
+
+### Extended Format
+
+The `extended` format is mainly intended for use by non-human consumers, like shell
+scripts or frontend code. It consists of an object with four top-level keys:
+
+ 1. `package_definition`: The original package definition object as found in the library.
+ 2. `resolved_package`: The effective package definition that was produced by applying
+    all inputs and overrides from the `config` section and removing all disabled pipelines
+    and contexts.
+ 3. `config`: The user-provided `config` object that was provided when installing the package.
+ 4. `package_status`: Contains additional information about the package provided by the package
+    manager, currently the `install_state` field that can be one of `installing`, `installed` or
+    `zombie` and the `from_configuration` boolean that shows whether a package was added
+    at runtime via the `package add` operator or from a configuration file on the node.
```

Comment:
The operator documentation follows a pretty strict format so that users know what to expect when opening the docs page for an operator. This deviates from that. I think specifying the possible output schemas as precisely as possible is nice, and we do this already for many other operators like `metrics` or `version`, so we could probably use them as a template.

Reply:
@lava I unresolved this, this should really be under the header `## Schemas` so that it matches the sections of the other operator docs.